docker/docker-compose.yml

```diff
@@ -82,12 +82,29 @@ services:
       container_name: lnd_ltc
       links:
           - "ltcd:blockchain"
+      volumes:
+          - lnd_ltc_data:/root/.lnd
```

Comment:
The volume shouldn't be required here if you've already defined it in the `Dockerfile` I believe.  Is there any reason why you're also setting it up here?

Reply:
I am mirroring the way volumes are set up for the other services.
e.g. - litecoin:/data
Also, I prefer having a named volume for ease of management.

Reply:
After doing some further testing it seems `docker-compose` requires this named volume for management _instead_ of an anonymous volume which is what's created in the `Dockerfile`.